specs/tree-view/tree-view.md

```diff
@@ -0,0 +1,186 @@
+- [Windows (UWP)](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tree-view)
+
+---
+
+## Design
+
+### API
+
+*The key elements of the component's public API surface:*
+
+**Tree View**
+
+- extends HTML Element props
+- custom CSS property for `depth` to support the infinite nesting scenario
+
+**Tree View Item**
+
+- expanded - nodes with child-nodes can either be expanded or collapsed. 
+- selected - the node is selected
+- content - slot for the content (the default slot for the item)
+- before-content - slot which precedes content, often a glyph or icon
+- after-content - slot which follows content, often a glyph or icon
+- expand-collapse-glyph - slot for svg, will include a default svg
+    - You could use a single glyph and rotate using CSS which would be the default. 
+    - With the expanded callback an implementor knows when a tree item is expanded or not. The glyph provided could be updated based on the expanded/collapsed state of the control.
+    - The glyph could be overridden with CSS using the named :part. This can change based on the expanded state as well.
+- expanded-change (event) - event for when the expanded state has changed
+- selected-change (event) - event for when an item has been selected
+
+### Anatomy and Appearance
+
+**Tree View**
+
+![](./images/tree-view.png)
+
+The tree view has no named slots or parts - it has a default slot for children.
+
+Flat tree:
+```html
+<fast-tree-view>
+    <fast-tree-item>Root node one</fast-tree-item>
+    <fast-tree-item>Root node two</fast-tree-item>
+    <fast-tree-item>Root node three</fast-tree-item>
+    <fast-tree-item>Root node four</fast-tree-item>
+</fast-tree-view>
+```
+
+Nested tree:
+```html
+<fast-tree-view>
+    <fast-tree-item>Root node one</fast-tree-item>
+    <fast-tree-item>Root node two</fast-tree-item>
+    <fast-tree-item>
+        <fast-tree-item>Nested node one</fast-tree-item>
+        <fast-tree-item>Nested node two</fast-tree-item>
+    </fast-tree-item>
+    <fast-tree-item>Root node three</fast-tree-item>
+</fast-tree-view>
+```
+
+**Tree view item**
+
+| State | Image |
+| ----- | ----- |
+| default | ![](./images/tree-item.png) |
+| selected | ![](./images/tree-item-selected.png)
+| expaned | ![](./images/tree-item-expanded.png)
+| collapsed | ![](./images/tree-item-collapsed.png)
+| selected/expanded | ![](./images/tree-item-selected-expanded.png)
+| selected/collapsed | ![](./images/tree-item-selected-expanded.png)
+
+Like tree view, the tree view item has a default slot for its children.
+
+Slots:
+- `content` - the default slot.
+- `child-nodes` - named slot for child nodes.
+- `before-content` - named slot to optionally include content (often a glyph) before the `content` slot
+- `after-content` - named slot to optionally include content (often a glyph) after the `content` slot
+- `expand-collapse-glyph` - named slot which can be used to provide an element which will serve as the expand/collapse glyph
```

Comment:
Seems a little duplicative with the slot list above.

Reply:
@EisenbergEffect the spec seems to provide multiple places for us to detail out slots. Slots, API, etc. Is one place preferred over the other?